File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/auth/dtos/RegisterUserResponse.java
package com.slotify.backend.spring.auth.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.util.UUID;

@Builder
@Getter
@ToString
public class RegisterUserResponse {

    private UUID userId;
    private String username;
    private String email;
}

File: app/src/main/java/com/raj/slotify/dtos/client/RegisterUserResponse.kt
package com.raj.slotify.dtos.client

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class RegisterUserResponse(
    @SerializedName("userId")
    val userId: UUID,

    @SerializedName("username")
    val username: String,

    @SerializedName("email")
    val email: String
)